pages/jobs/[jobName].tsx

```diff
@@ -21,6 +22,23 @@ const STATES = {
 	},
 };
 
+/**
+ * let myAdd: (x: number, y: number) => number = function (
+  x: number,
+  y: number
+): number {
+  return x + y;
+};
+ */
```

Comment:
I'm just gonna pull your PR down and remove this portion and squash into your commit. I will re-send in a new PR